ingest/scripts/standardize-lineage

```diff
@@ -13,6 +13,7 @@ from augur.io.print import print_err
 # GISAID lineage patterns copied from nextstrain/fauna/vdb/flu_upload.py
 # <https://github.com/nextstrain/fauna/blob/fbdc393581b1859ac817403d8f43e114d7edbc60/vdb/flu_upload.py#L22-L44>
 GISAID_LINEAGE_MAP = {
```

Comment:
[tangential] I remember this from avian-flu fauna - when I first saw it I had to do a bunch of digging to work out what the different tuples meant and how they worked, and seeing it now I would have to do exactly the same digging since I've completely forgotten. It'd be great to add some commentary about what's going on here.

Reply:
Added in https://github.com/nextstrain/seasonal-flu/pull/236/commits/ac2b5496cc6ff2c3ea76a777de6b84e7ee833719. Let me know if you want more clarification here.